packages/smooth_app/lib/pages/product/new_product_page.dart

```diff
@@ -264,6 +264,10 @@ class _ProductPageState extends State<ProductPage>
             ),
           ),
         ),
+        margin: const EdgeInsets.symmetric(
+          horizontal: SMALL_SPACE,
+          vertical: MEDIUM_SPACE
```

Comment:
I don't think we really care about the website widget in itself. We care about the last item of the list. Or rather about the list itself.